app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -1121,6 +1122,39 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_clickforgotPin_enterAdminPin_clickSubmit_dialogMessageIsCorrect() {
```

Comment:
```suggestion
  fun testPinPassword_clickForgotPin_enterAdminPin_clickSubmit_dialogMessageIsCorrect() {
```

I missed this the first time around.

Reply:
@BenHenning 
https://github.com/oppia/oppia-android/pull/4487#discussion_r953411243
Added this to maintain flow of the test.

Reply:
@BenHenning Updated the test name.